comp/core/secrets/secretsimpl/fetch_secret_test.go

```diff
@@ -249,9 +249,7 @@ func TestFetchSecret(t *testing.T) {
 		"handle2": "p2",
 	}, resp)
 	assert.Equal(t, map[string]string{
-		"test":    "yes",
-		"handle1": "p1",
-		"handle2": "p2",
```

Comment:
Are these changes expected?

Reply:
Yes, before `fetchSecret` was storing results in the cache. But this change is moving the cache writes up into `resolve`, but the test is directly using `fetchSecret`.

Reply:
Thanks for the clarification